aws/hll/aws-s3-transfer-manager/examples/cp.rs

```diff
@@ -101,7 +104,7 @@ fn invalid_arg(message: &str) -> ! {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
-    tracing_subscriber::fmt::init();
+    console_subscriber::init();
```

Comment:
Are these mutually exclusive?

Reply:
Yeah I believe so. IIRC `console_subscriber` effectively sets up the equivalent as `tracing_subscriber::fmt::init()`. Plan on playing with it more as I could see us wanting a flag to enable/disable console subscriber.